.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: python
 
-script:
+before_script:
```

Comment:
Is there a reason to do this as a before_script vs. just having multiple listed script entries?

Reply:
stages could also be useful: https://docs.travis-ci.com/user/build-stages/

Reply:
it should just be in `script`, I misread. `before_script` is for stuff that the computers take care of (loading deps, etc) that would show a broken build instead of a failed build that requires code fixes